src/main/java/de/jeff_media/InvUnload/Main.java

```diff
@@ -30,7 +32,7 @@ public class Main extends JavaPlugin implements Listener {
 	int mcMinorVersion; // 14 for 1.14, 13 for 1.13, ...
 
 	@SuppressWarnings("FieldCanBeLocal")
-	private final int currentConfigVersion = 34;
+	private final int currentConfigVersion = 37;
```

Comment:
The new config version is set to 37 instead of just incremented by one because the latest version ([5.0.2](https://www.spigotmc.org/resources/invunload.60095/update?update=474801)) published on spigot.org has it set to 36. It looks like the code in repo doesn't contain changes from the v5.0.2.